RedactedScript: Version 1.001; ttfautohint (v1.8.3) added

emmamarichal opened the pull request:
65fb96d: [gftools-packager] RedactedScript: Version 1.001; ttfautohint (v1.8.3) added

* RedactedScript Version 1.001; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/christiannaths/redacted-font at commit https://github.com/christiannaths/redacted-font/commit/dae1afaa7d96c89825c5e0193f07b4b16ccef4d7.

 29bb40a: [gftools-packager] ofl/redactedscript remove METADATA "source".  google/fonts#2587

RosaWagner commented:
@emmamarichal check your yaml file and see if you didn't forget a space in the name of the font

emmamarichal commented:
Updated

## Redacted Script: Version 1.001; ttfautohint (v1.8.3) added

---
 9be995e: [gftools-packager] Redacted Script: Version 1.001; ttfautohint (v1.8.3) added

* Redacted Script Version 1.001; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/christiannaths/redacted-font at commit https://github.com/christiannaths/redacted-font/commit/dae1afaa7d96c89825c5e0193f07b4b16ccef4d7.

 ee0da9c: [gftools-packager] ofl/redactedscript remove METADATA "source".  google/fonts#2587
